assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.scss

```diff
@@ -74,9 +74,10 @@
   height: 40px;
   width: 40px;
   float: right;
+  display: flex;
```

Comment:
The fact that the cross is correctly centered on the existing pages is just a random consequence of the font-size of the container, and it wasn't centered on the new page. So I've used `flex` to center the button, everywhere

Reply:
I use flex _all the time™_

Reply:
yep... although I just discovered some caveats which affect browsers we support. and this specific case, where I'm applying `display: flex` to a `<button></button>`. https://github.com/philipwalton/flexbugs#9-some-html-elements-cant-be-flex-containers

Reply:
Fix on the way....